### PR TITLE
Updated code to pass unit/integration tests

### PR DESCRIPTION
**Pull request by SaminSarker05**
- added unit and integration tests under tests folder at root
- used peewee and unittest frameworks for testing with a in memory db

- updated POST api endpoint to check for invalid or missing payload fields

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -85,9 +89,12 @@ def timeline():
 
 @app.route("/api/timeline_post", methods=["POST"])
 def post_time_line_post():
+    if "name" not in request.form: return "Invalid name", 400
     name = request.form["name"]
     email = request.form["email"]
+    if "@" not in email and ".com" not in email: return "Invalid email", 400
```
Comment (SaminSarker05) [inline app/__init__.py:95]: although better methods prob exist for email checking i used this as a bandaid solution

Comment (SaminSarker05) [inline app/__init__.py:95]: should be OR instead of AND, my bad!

Comment (kumankai) [reply app/__init__.py:95]: got it

**Comment by kumankai** [comment]
Good!

**Pull request merged**